packages/shared/components/popups/LedgerLegacyTransaction.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+    import { Illustration, Text } from 'shared/components'
+    import { formatUnitBestMatch } from 'shared/lib/units'
+
+    type Output = {
+        address: string
+        value: number
+        checksum: string
+    }
+    type Input = {
+        address: string
+        balance: number
+        index: number
+        checksum: string
+    }
+
+    export let locale
+    export let output: Output
+    export let inputs: Input[]
+
+    const formatAmount = (amountRaw: number): string => {
```

Comment:
I see a similar method here https://github.com/iotaledger/firefly/pull/1246/files#diff-779dc1ec1c8db9e429fe726b464b33b7e906c463e66ee937c786127d40a3f7d8R55-R57. I think it is best to have it merged or maybe the abstraction over the `formatUnitBestMatch` is redundant.

Reply:
Redundant, removing